pkg/network/kuryr_bootstrap.go

```diff
@@ -0,0 +1,851 @@
+package network
```

Comment:
Minor organizational request: Can you move this to `pkg/platform/openstack/`? I think `pkg/network` is getting too big.